api/config/service/authn/v1/authn.proto

```diff
@@ -14,6 +14,8 @@ message OIDC {
   string redirect_url = 4 [ (validate.rules).string = {min_bytes : 1} ];
 
   repeated string scopes = 5;
+
+  string subject_claim_name_override = 6;
```

Comment:
can you document the default subject claim when it is not overridden just above this line?